src/SmartComponents/Remediations/RemediationsCard.js

```diff
@@ -68,6 +68,20 @@ const RemediationsCard = () => {
         </TemplateCardHead>
         <TemplateCardBody>
             {remediationsList}
+            <div className="ins-c-remediations-container">
+                <div className="ins-c-remediation__status">
+                </div>
+                <div className="ins-c-remediation__timestamp">
+                    <Button
+                        component="a"
+                        href=""
+                        variant="link"
+                        isInline
+                    >
+                        more remediations
```

Comment:
Need to build this string out in messages.js, gotta keep up with translations 🦹‍♀️